docs/pages/guides/publishing-websites.mdx

```diff
@@ -1,17 +1,26 @@
 ---
-title: Publish websites with Metro
-description: Different ways to publish the Expo web app with third-party services.
+title: Publish websites
+description: Learn how to deploy Expo websites for production.
 ---
 
 import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
 import { YesIcon, NoIcon } from '~/ui/components/DocIcons';
 import { Terminal } from '~/ui/components/Snippet';
 import { Step } from '~/ui/components/Step';
 import { Tab, Tabs, TabsGroup } from '~/ui/components/Tabs';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { Rocket02Icon } from '@expo/styleguide-icons/outline/Rocket02Icon';
 
-> **warning** In SDK 50 and above, `metro` is the default bundler for web apps. The previous [guide for publishing with `webpack` is available here](/archive/publishing-websites-webpack/).
+An Expo web app can be served locally for testing the production behavior, and deployed to a hosting service. We recommend deploying to [EAS Hosting](/eas/hosting) for the best feature support. You can also self-host or use a third-party service.
 
-A web app created using Expo can be served locally for testing out the production behavior. Once the testing phase checks out, you can choose from a variety of third-party services to host it.
+<BoxLink
+  title="Deploying instantly with EAS"
+  description="EAS Hosting is the best way to deploy your web app with support for custom domains, SSL, and more."
+  href="/eas/hosting/get-started"
+  Icon={Rocket02Icon}
```

Comment:
```suggestion
  Icon={Cloud01Icon}
```

This icon should represent the service itself. Let's use `Cloud01Icon` icon here for EAS Hosting. You can import it like below:

```
import { Cloud01Icon } from '@expo/styleguide-icons/outline/Cloud01Icon'
```